src/Detail/Commanding/Command/Listing/Filter.php

```diff
@@ -116,4 +118,37 @@ public function setType($type)
     {
         $this->type = $type;
```

Comment:
we have 2 possibilities here:
- work with a raw_value
- or change visibility of setType to private